src/main/java/com/kodilla/ecommercee/exception/ProductNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.ecommercee.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST, reason = "Wrong product id!")
+public class ProductNotFoundException extends RuntimeException {
```

Comment:
not found to 404, bad request jest wtedy kiedy np dane formularza sa niepoprawne